File: src/test/ironfist/items/RingDescriptionResourcesTest.java
package ironfist.items;

import java.text.MessageFormat;
import java.util.Enumeration;
import java.util.ResourceBundle;

import junit.framework.TestCase;

/**
 * DOCUMENT ME!
 * 
 * @author pmgremo
 */
public class RingDescriptionResourcesTest extends TestCase {

  public void testLoadResource() throws Exception {
    ResourceBundle bundle = ResourceBundle.getBundle("ironfist.items.RingDescriptionResources");
    Enumeration<String> keys = bundle.getKeys();

    keys = bundle.getKeys();

    String pattern = bundle.getString(keys.nextElement());
    System.out.println(MessageFormat.format(pattern,
      new Object[]{new Integer(0)}));
    System.out.println(MessageFormat.format(pattern,
      new Object[]{new Integer(1)}));
    System.out.println(MessageFormat.format(pattern, new Object[]{new Integer(
      30)}));
  }
}

File: src/test/java/ukalus/items/RingDescriptionResourcesTest.kt
package ukalus.items

import junit.framework.TestCase
import java.text.MessageFormat.format
import java.util.*

/**
 * DOCUMENT ME!

 * @author pmgremo
 */
class RingDescriptionResourcesTest : TestCase() {

    fun testLoadResource() {
        val bundle = ResourceBundle.getBundle("ukalus.items.RingDescriptionResources")
        val pattern = bundle.getString(bundle.keys.nextElement())
        println(format(pattern, 0))
        println(format(pattern, 1))
        println(format(pattern, 30))
    }
}